xactserver/src/bin/xactserver.rs

```diff
@@ -30,11 +36,16 @@ fn main() -> anyhow::Result<()> {
 
     let listen_pg = args.value_of("listen-pg").unwrap_or("127.0.0.1:10000");
     let listen_peer = args.value_of("listen-peer").unwrap_or("127.0.0.1:23000");
+    let connect_pg = args.value_of("connect-pg").unwrap_or("127.0.0.1:5432");
     let peers = args.values_of("peers").unwrap_or_default().collect();
 
+    // Create a pg dispatcher to dispatch received remote txn
+    let (remotexact_tx, remotexact_rx) = mpsc::channel(100);
+    let pg_dispatcher = PgDispatcher::new(connect_pg);
+   
```

Comment:
Move these lines to under the `pg_watcher` definition below. 

Rename `(remotexact_tx, remotexact_rx)` to `(remote_log_tx, remote_log_rx)`